$NetBSD$

Avoid following warning at make package-install:
warning: share/dejagnu/baseboards/am33_2.0-libremote.exp: executable bit is set on non-executable file.

--- Makefile.am.orig	2011-03-09 09:47:22.000000000 +0900
+++ Makefile.am	2013-06-07 23:42:03.000000000 +0900
@@ -33,15 +33,17 @@ bin_SCRIPTS = runtest
 include_HEADERS = dejagnu.h
 
 pkgdata_SCRIPTS = \
-  runtest.exp \
+  runtest.exp
+
+pkgdata_DATA = \
+  stub-loader.c testglue.c \
   lib/debugger.exp lib/dejagnu.exp lib/dg.exp lib/framework.exp \
   lib/ftp.exp lib/kermit.exp lib/libgloss.exp lib/remote.exp \
   lib/rlogin.exp lib/rsh.exp lib/standard.exp lib/target.exp \
   lib/targetdb.exp lib/telnet.exp lib/tip.exp lib/utils.exp
 
 configdir = $(pkgdatadir)/config
-config_DATA = config/README
-config_SCRIPTS = \
+config_DATA = config/README \
 	config/arc.exp \
 	config/arm-ice.exp \
 	config/base-config.exp \
@@ -84,8 +86,7 @@ config_SCRIPTS = \
 	config/vxworks.exp
 
 baseboarddir = $(pkgdatadir)/baseboards
-baseboard_DATA = baseboards/README
-baseboard_SCRIPTS = \
+baseboard_DATA = baseboards/README \
 	baseboards/am33_2.0-libremote.exp \
 	baseboards/arc-sim.exp \
 	baseboards/arm-ice.exp \
@@ -184,8 +185,6 @@ baseboard_SCRIPTS = \
 	baseboards/xscale-cygmon.exp \
 	baseboards/xtensa-sim.exp
 
-pkgdata_DATA = stub-loader.c testglue.c
-
 djlibexecdir = $(pkgdatadir)/libexec
 djlibexec_SCRIPTS = config.guess
 
